src/database/services/severa-api-service.ts

```diff
@@ -1,23 +1,32 @@
 import fetch from "node-fetch";
 import type Flextime from "../models/severa";
+import * as process from "node:process";
```

Comment:
I don't think this import is necessary and can be removed. I've created another task in trello for ensuring envs are imported using envalid but as there are multiple other examples of envs being imported in similar way you are this can be a task on it's own. Feel free to have a look at it if you want but no need for the changes to be done in this PR